backend/app/services/rag/index/indexer.py

```
@@ -154,22 +185,217 @@ def index_from_binary(
+
+        # Create temporary file for DoclingReader
         with tempfile.NamedTemporaryFile(
             suffix=file_extension, delete=False
         ) as tmp_file:
             tmp_file.write(binary_data)
             tmp_file_path = tmp_file.name
 
         try:
+            # General documents use Markdown export format
+            export_type = DoclingReader.ExportType.MARKDOWN
+            logger.info(f"Configuring DoclingReader with Markdown export for general document")
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)